motionplan/ik/combinedInverseKinematics.go

```diff
@@ -15,22 +16,23 @@ import (
 
 // CombinedIK defines the fields necessary to run a combined solver.
 type CombinedIK struct {
-	solvers []InverseKinematics
-	model   referenceframe.Frame
-	logger  logging.Logger
+	solvers    []InverseKinematics
+	logger     logging.Logger
+	lowerBound []float64
```

Comment:
We don't want `[][]float64`, because `ik` still operates on arrays of floats; it is simply a generic function rather than specifically a Frame on which Transform is called.

The overall motivation is to make it easy for `motionplan` to solve for framesystem states, yes. But that is not the same thing as actually passing framesystems into `ik`, that's not what we want. We just want `ik` here to be a generic function, so that we may create a distance function which measures a frame system, a single frame, or something else unrelated to frames.

Reply:
How about an array of referenceframe.Limit here instead?

Reply:
That's what we instantiate with, but we need to turn it into floats to actually pass into the C `*nlopt` structure